packages/docs/src/routes/api/qwik-server/index.md

```diff
@@ -118,7 +118,7 @@ export interface QwikLoaderOptions
 
 | Property       | Modifiers | Type                          | Description  |
 | -------------- | --------- | ----------------------------- | ------------ |
-| [events?](#)   |           | string\[\]                    | _(Optional)_ |
+| [events?](#)   |           | string[]                      | _(Optional)_ |
```

Comment:
Is this correct? Shouldn't this stay the same since it's not a link?

Reply:
thanks for the review @wmertens 🙏 it shouldn't affect the output though. it's escaped by the api extractor but not needed to be imo on an .md file. do you see an issue with it? thanks again for your input 😃